app/models/invoice.rb

```diff
@@ -0,0 +1,3 @@
+class Invoice < ApplicationRecord
+  enum status: { 'in progress' => 0, 'completed' => 1, 'cancelled' => 2 }
```

Comment:
It looks like these are a more dynamic version of the adopt don't shop status thing. So it's easier to change a status or display only 'in progress' invoices, stuff like that. Cool!